pull-request/src/person.js

```diff
@@ -12,4 +12,5 @@ Person.prototype.fullname = function() {
 
 Person.prototype.wishHappyBirthday = function() {
     return "Happy birthday " + this.firstName;
+    this.age++;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this code after the "return" statement. ([javascript:UnreachableCode](http://localhost:9750/coding_rules#rule_key=javascript%3AUnreachableCode))

[See it in SonarQube](http://localhost:9750/project/issues?id=training%3Apull-request&pullRequest=39&issues=AWgUBXbJ6PH_c9BW2dOB&open=AWgUBXbJ6PH_c9BW2dOB)

[ISSUE_KEY]: AWgUBXbJ6PH_c9BW2dOB